Fixed tasks in Login pages and DCL Landing

HenryGBC opened the pull request.

Modulos commented:
Hei @HenryGBC could you please provide some information about what exactly do you fix? Also since this seems to include visual changes can you please add some screenshots of your changes?

HenryGBC commented:
In Commit "Fixed details in Hosting pages and translate in login"
Fixed details in "dashboard VM"(Hosting pages)

- in /my-virtual-machines : Centered content, and center fields in table
- in /my-virtual-machines/:id : Fixed blank space in status, and fixed height boxes 
- in /orders : Centered content, and center fields in table
- in /orders:id : Centered content, and fixed invoice spaces
- in /vm-key-pair: Centered content, and center fields in table


In commit "Collapse navbar when i clicked url dcl"
Hide navbar when i clicked somethimg url..

In Commit "Fix center login pages"

Centered login, signup, and reset password boxes and not scroll in mobile..

HenryGBC commented:
In commit "improveded mobile responsiveness"
Fixed Issue #3271 in Redmine; Mobile responsiveness in DCL Landing, Calculator Card and Login(Hosting pages)

-Change Text in section What is it? and Fixed Typo
